packages/trip-form/i18n/en-US.yml

```diff
@@ -52,3 +52,29 @@ otpUi:
     use: Use # as in "Use bus, train, subway".
     useCompanies: Use companies # as in :"Use companies: Spin, Lime, Bolt"
     walkOnly: Walk Only
+  ModeSelector:
+    labels:
+      TRANSIT: Transit
+      BICYCLE: Bike
+      WALK: Walk
+      CAR: Drive
+      RENT: Rent
+    settings:
+      walkTolerance-labelLow: Less walking
+      walkTolerance-labelHigh: More walking
+      walkTolerance-label: Walk Tolerance
+      bikeTolerance-labelLow: Less biking
+      bikeTolerance-labelHigh: More biking
```

Comment:
Capitalize this text too? (the majority of the other ones seem to be capitalized already).